src/components/withWindowDimensions/index.js

```diff
@@ -1,9 +1,10 @@
-/* eslint-disable react/no-unused-state */
 import React, {forwardRef, createContext} from 'react';
```

Comment:
What do you think of moving `getWindowHeightAdjustment.js` to `src/libs/getWindowHeightAdjustment/index.js` and `.../index.android.js`, and keeping this as `withWindowDimensions.js` (instead of moving to `withWindowDimensions/index.js`)

Reply:
That could work too, although I think this function is very specific to `withWindowDimensions` and should not be used anywhere else, this is why I think I prefer keeping it in the same folder.